offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/affiliation/update/UpdateAffiliationView.groovy

```diff
@@ -132,4 +144,82 @@ class UpdateAffiliationView extends FormLayout implements Resettable, Updatable<
       }
     }
   }
+
+  /**
+   * Modal window which prompts the user to confirm that the affiliation category change was intended
+   * @param Affiliation affiliation the to be updated affiliation for which the category was changed
+   */
+
+  private class ConfirmAffiliationChangeWindow extends Window {
+    private Button abortButton
+    private Button submitButton
+    private Affiliation updatedAffiliation
+
+    ConfirmAffiliationChangeWindow(Affiliation affiliation) {
+      this.updatedAffiliation = affiliation
+      generateAffiliationChangeWindow()
+    }
+
+    private void generateAffiliationChangeWindow() {
+      this.setCaption("Confirm affiliation update")
+      this.setContent(generateWindowContent())
+      setupWindowStyle()
+      registerListeners()
+    }
+
+    private VerticalLayout generateWindowContent() {
+      VerticalLayout windowContent = new VerticalLayout()
+      windowContent.addComponents(generateText(), generateButtonRow())
+      return windowContent
+    }
+
+    private Label generateText() {
+      Label windowText = new Label("Changing the affiliation category affects the pricing of <b> ALL </b> offers associated with this affiliation!")
+      windowText.setContentMode(ContentMode.HTML)
+      return windowText
+    }
+
+    private void setupButtons() {
+      this.abortButton = new Button("Abort Affiliation Update")
```

Comment:
Maybe the text here is misleading as the customer does not abort the affiliation update but rather decides to go back to editing. 
```suggestion
      this.abortButton = new Button("Back to editing!")
```